archive/Thematic/2021/Februrary/progressed/bridgecrew2/template.html

```diff
@@ -790,7 +816,17 @@ <h1 class="null" style="text-align: center;display: block;margin: 0;padding: 0;c
 																style="padding: 0px 18px 9px;line-height: 150%;mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;word-break: break-word;color: #111111;font-family: 'Source Sans Pro', 'Helvetica Neue', Helvetica, Arial, sans-serif;font-size: 18px;text-align: left;">
 
-																XXXX
+																<ol start="4">
+																	<li><strong style="font-weight: bolder;">DevOps Requires Teams’ Physical Proximity</strong></li>
+																</ol>
+																<div dir="ltr" style="text-align: justify;"><span style="font-size:16px"><span style="font-family:trebuchet ms,lucida grande,lucida sans unicode,lucida sans,tahoma,sans-serif">If you still have this delusion after months of lockdown, you need <a href="https://hackernoon.com/github-actions-and-go-areyouok-my-url-ao1d33a1" target="_blank" style="mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;color: #111111;font-weight: bold;text-decoration: underline;">a good dose of reality</a>. With the right tools and frameworks to promote communication and collaboration along the DevOps lifecycle, <a href="https://hackernoon.com/a-solution-to-your-mental-health-worries-think-big-towards-your-future-and-the-rest-will-follow-flbk33gp" target="_blank" style="mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;color: #111111;font-weight: bold;text-decoration: underline;">remote workplace culture</a> can go toe-to-toe with in-office teams. Besides, remote teams usually have greater visibility into what others are working on, which also supports the main idea of DevOps.</span></span></div>
+																<br/>
+																<div dir="ltr" style="text-align: justify;"><span style="font-size:16px"><span style="font-family:trebuchet ms,lucida grande,lucida sans unicode,lucida sans,tahoma,sans-serif"><p dir="ltr" style="text-align: center;line-height: 150%;margin: 10px 0;padding: 0;mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;color: #111111;font-family: 'Source Sans Pro', 'Helvetica Neue', Helvetica, Arial, sans-serif;font-size: 18px;"><span style="font-family:georgia,times,times new roman,serif"><span style="font-size:17px"><a href="https://media.giphy.com/media/eYilisUwipOEM/giphy.gif" target="_blank" style="mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;color: #111111;font-weight: bold;text-decoration: underline;"><img data-file-id="1041068" src="https://gitlab.com/hackernoon/creative/-/raw/master/newsletters/memes/2021/Feb/16.02/image3.gif" style="border: 0px initial;width: 220px;height: 134px;margin: 0px;outline: none;text-decoration: none;-ms-interpolation-mode: bicubic;" alt="visit hackernoon.com"></a></span></span></p></span></span></div>
+																<br/>
+																<ol start="5">
+																	<li><strong style="font-weight: bolder;">DevOps Guns for ITIL</strong></li>
```

Comment:
Same issue

Reply:
use this https://github.com/atherdon/newsletters/tree/master/archive/Thematic/snippets#h2